setup.py

```diff
@@ -36,7 +36,7 @@
             'pytest-aiohttp',
             'pytest-cov',
             'coverage>=4.4',
-            'pytest-docker-fixtures[rabbitmq]==1.2.5',
+            'pytest-docker-fixtures[rabbitmq]==1.2.7',
```

Comment:
It's not a good idea to use hard-pins in setup.py--you're forcing consumers to use this.

Can you change it to `>=`?